website/content/docs/configuration/session-recording/create-storage-bucket.mdx

```diff
@@ -314,6 +315,84 @@ Complete the following steps to create a storage bucket in Boundary.
+   - **Disable credential rotation**: (Optional) Controls whether the plugin will rotate the incoming credentials and manage a new storage service account. If this attribute is set to false, or not provided, the plugin will rotate the incoming credentials, using them to create a new storage service account, then delete the incoming credentials.
+
+     Note that credential rotation is not supported for Hitachi Content Platform, and it may not function for other S3-compatible providers.
+
+1. Click **Save**.
+
+</Tab>
+<Tab heading="CLI">
+
+1. Log in to Boundary.
+1. Use the following command to create a storage bucket in Boundary:
+
+   ```shell-session
+   $ boundary storage-buckets create \
+      -bucket-name mystoragebucket \
+      -plugin-name minio \
+      -scope-id o_1234567890 \
+      -bucket-prefix="foo/bar/zoo" \
+      -worker-filter '"dev" in "/tags/type"' \
+      -attr endpoint_url="https://my-hitachi-instance.dev:9000" \
+      -attr region="REGION" \
+      -attr disable_credential_rotation=true \
+      -secret access_key_id="KEY" \
+      -secret secret_access_key="SECRET"
+   ```
+
+   Replace the values above with the following required secrets and any optional [attributes](/boundary/docs/concepts/domain-model/storage-buckets) you want to associate with the Boundary storage bucket:
+
+   - `bucket-name`: (Required) Name of the S3-compliant storage bucket you want to associate with the Boundary storage bucket.
+   - `plugin-name`: (Required) The name of the Boundary storage plugin.
+   Use the `minio` plugin for S3-compatible storage.
+   - `scope_id`: (Required) A storage bucket can belong to the Global scope or an Org scope.
+   - `worker-filter`: (Required) A filter that indicates which Boundary workers have access to the storage. The filter must match an existing worker in order to create a Boundary storage bucket.
+   - `secret`: (Required) The storage provider's credentials to use.
+      - `access_key_id` (Required): The storage provider's service account's access key to use with this storage bucket.
+      - `secret_access_key` (Required): The storage provider's service account's secret key to use with this storage bucket.
+   - `attributes` or `-attr`: Attributes of the S3-compliant storage bucket.
+      - `endpoint_url` (Required): Fully-qualified endpoint pointing to an S3-compliant API. This example uses Hitachi, but you should substitute your storage provider's endpoint.
+      - `region`: (Optional) The region to configure the storage bucket for.
+      - `disable_credential_rotation`: (Optional) Controls whether the plugin will rotate the incoming credentials and manage a new storage service account. If this attribute is set to `false`, or not provided, the plugin will rotate the incoming credentials to create a new storage service account, then delete the incoming credentials.
+
+      This option must be set to `true` if you use dynamic credentials.
```

Comment:
We only support dynamic credentials for AWS - Other services don't really have this at all, so I think we could get rid of this line given the context in this page?

```suggestion

```